### PR TITLE
Update the statement You can only reference a user by the ACCOUNT_ID in an owner file in Doc of Pipelines as Code with Bitbucket Server

### PR DESCRIPTION
**Pull request by PoornimaSingour**
Update the statement You can only reference a user by the ACCOUNT_ID in an owner file in Doc of Pipelines as Code with Bitbucket Server

Version(s):
 pipelines-docs-1.13, pipelines-docs-1.14

Issue:

https://issues.redhat.com/browse/OCPBUGS-29885

Link to docs preview:

https://docs.openshift.com/pipelines/1.11/pac/using-pipelines-as-code-repos.html#using-pipelines-as-code-with-bitbucket-server_using-pipelines-as-code-repos

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/op-using-pipelines-as-code-with-bitbucket-server.adoc [hunk 1]
```diff
@@ -88,7 +88,7 @@ spec:
       key: "webhook.secret" # Set this if you have a different key for your secret
 ----
 <1> Ensure that you have the right Bitbucket Server API URL without the `/api/v1.0` suffix. Usually, the default install has a `/rest` suffix.
-<2> You can only reference a user by the `ACCOUNT_ID` in an owner file.
+<2> Specify the BitBucket Server username.
```
Comment (agantony) [inline modules/op-using-pipelines-as-code-with-bitbucket-server.adoc:91]:
```suggestion
<2> Specify the username of the BitBucket Server.
```

**Review by agantony — COMMENTED** [review]
Added a suggestion for your consideration; overall lgtm!

Preview URL?

**Review by agantony — COMMENTED** [review]
Peer review is done.

**Comment by PoornimaSingour** [comment]
Hi @agantony , Thank you for the help, I have updated all the details suggested and also updated the Preview URL. Could you please help further on this?

**Review by sheriff-rh — APPROVED** [review]
LGTM, merging.

**Pull request merged**